### DISCUSSION TITLE
Tab

### DISCUSSION THREAD

**Discussion by cheadevit** [Laravel-Backpack/community-forum#382] [Q&A (Help)] [answered]
Hello guys, I have a question, I tried to use the tab which separates into 2 tabs and points to the same function model, but it shows only 1 tab. 
here is my code:
```
    public function itemCheck()
    {
        return $this->belongsToMany('App\Models\Item', 'checked_details')
        ->withPivot(['status_id'])->withTimestamps();
    }
```

```
        $this->crud->addField([
            'label'     => 'Check Detail',
            'type'      => 'checklist',
            'name'      => 'CheckedVehicle.itemCheck',
            'entity'    => 'CheckedVehicle.itemCheck',
            'attribute' => 'name',
            'model'     => "App\Models\Item",
            'pivot'     => true,
            'number_of_columns' => 1,
            'inline'   => true,
            'wrapper' => [
                'class' => 'col-md-12'
            ],
            'options' => (function ($query) {
                return $query->where('cat_id', 1)->pluck('name', 'id');
            }),
            'tab' => 'Part 1'
        ]);

        $this->crud->addField([
            'label'     => 'Check Detail',
            'type'      => 'checklist',
            'name'      => 'CheckedVehicle.itemCheck',
            'entity'    => 'CheckedVehicle.itemCheck',
            'attribute' => 'name',
            'model'     => "App\Models\Item",
            'pivot'     => true,
            'number_of_columns' => 1,
            'inline'   => true,
            'wrapper' => [
                'class' => 'col-md-12'
            ],
            'options' => (function ($query) {
                return $query->where('cat_id', 2)->pluck('name', 'id');
            }),
            'tab' => 'Part 2'
        ]);
```

![Screen Shot 2023-02-01 at 11 23 55 AM](https://user-images.githubusercontent.com/52439329/215947032-61c44d2a-2daa-4372-894b-3ced932a0cbc.png)

appreciate it if anyone could help, thank you.

**Comment by o15a3d4l11s2** [comment, marked as answer]
You cannot have one and the same field (with the same name) multiple times. The CRUD needs to be able to make a difference between the two, otherwise it is just like adding the same field twice, which just overwrites the field. I recommend checking [Accessors & Mutators](https://laravel.com/docs/9.x/eloquent-mutators#accessors-and-mutators). This way you can have a field name that is not a real database column and still read/write it.

For more details you can check these threads:
- https://stackoverflow.com/questions/56578029/backpack-for-laravel-display-two-fields-with-same-name
- https://github.com/Laravel-Backpack/CRUD/issues/1905
